packages/map_editor/maps/empty/main.yaml

```diff
@@ -4,10 +4,11 @@ main:
   tile_maps:       !include tile_maps.yaml
   watchtowers:     !include watchtowers.yaml
   citizens:        !include citizens.yaml
-  trafficsigns:    !include trafficsigns.yaml
-  groundtags:      !include groundtags.yaml
+  traffic_signs:   !include trafficsigns.yaml
+  ground_tags:     !include groundtags.yaml
+  vehicle_tags:    !include vehicletags.yaml
```

Comment:
The files are still called `trafficsigns`, `groundtags` and `vehicletags`, we need an underscore there too.